src/Components/Comments.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
```

Comment:
Make sure to remove these auto imports- vscode is trying to be helpful but failing :P